data/json/items/furniture.json

```
@@ -19,7 +19,7 @@
     "type": "GENERIC",
     "id": "armchair",
     "name": { "str": "armchair" },
-    "description": "A simple upholstered chair with armrests.  Soft and fairly warm, it definitely beats the ground for sleeping on, though not by much.",
+    "description": "A simple upholstered chair with armrests.  Soft and fairly warm, it definitely beats the ground for sleeping on, though not by much. Can be placed via the construction menu.",
```

```suggestion
    "description": "A simple upholstered chair with armrests.  Soft and fairly warm, it definitely beats the ground for sleeping on, though not by much.  Must be deployed upright to be used.",
```

Would this kind of language be better at maintaining 4th wall while also pointing out that something must be done to place it properly?  Also the item sprite could be drawn on it's side to represent the item not being upright.

---

We could state (somewhat reasonably) that the item-form is packed for transport, and therefore it "Must be unpacked to be used"? Or even, in keeping with other messages, "You need to unpack this to sit on it" -- speaking to the character, not the player.